### DISCUSSION TITLE
Bunch of questions

### DISCUSSION THREAD

**Discussion by Durahl** [supermerill/SuperSlicer#2165] [Q&A]
Greetings!

I've collected a few questions that have bothered me for a while now:

1. Is it possible to disable ( both **Globally** and **Layer Height** specifically ) the **Solid Infill** that is being generated at shallow surfaces?
![IMG](https://i.imgur.com/KdUylfD.png)
2. Is there a way to prevent [this kind of behaviour](https://youtu.be/ciWMIr8VOjQ) of the Print Head crossing the seven seas to get the chicken on the moon to cross the other side of the road just to draw two Perimeters and one Infill? That **Gap Infill** starting at the _exact opposite_ side of the two **Outer Perimeters** sure seems unnecessary.
3. A question regarding the merging of PS features into SS - Can it basically be assumed that _any_ feature that is being added to PS ( like all the new stuff in PS 2.4 ) will at some point find its way into SS or is this more of a selective thing where some things may not make the cut for whatever reason?

Cheers!

**Comment by blex-max** [comment]
Quick answer to i) - try raising 'No solid infill over' X 'perimeters' to 1, and turn off 'Ensure vertical shell thickness'. That _should_ remove the undesirable solid infill.

**Reply by supermerill** [reply]
For 1, trey with and without 'Ensure vertical shell thickness', I have to work a bit more with that and I may have say something wrong in the tooltip.

2 - The gcode writer has a list of things to extrude. If they can be reordered, it try to go the nearest point (choosing a seam if it's a loop). Can be improved.
Maybe put more weight to the "travel cost" of the seam,  or use "aligned"

3 - yes unless it's already superseded by something.
For example, I won't activate their new brim code as mine is still much more powerful.

**Comment by Durahl** [comment]
Okay, I'll give those suggestions a try next time I come across the issues =)
